code/client/src/core/dev_features.cpp

```diff
@@ -90,10 +90,13 @@ namespace MafiaMP::Core {
             ToggleVehicleDebug();
         }
 
+        static bool bTempLockControls = false;
```

Comment:
This is all temporary so I can test because ImGui cursor is currently broken. I'll likely remove it out from this PR when it has been approved.